src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -98,7 +98,7 @@ __vsc_update_prompt() {
 		# means the user re-exported the PS1 so we should re-wrap it
 		if [[ "$__vsc_custom_PS1" == "" || "$__vsc_custom_PS1" != "$PS1" ]]; then
 			__vsc_original_PS1=$PS1
-			__vsc_custom_PS1="\[$(__vsc_prompt_start)\]$PREFIX$__vsc_original_PS1\[$(__vsc_prompt_end)\]"
```

Comment:
@Tyriar do you know why this was needed / added before?

Reply:
I think this came across when switching to starship's method, so probably not needed